consultation_analyser/consultations/decorators/sagemaker_endpoint_status_check.py

```diff
@@ -0,0 +1,23 @@
+import boto3
+from django.conf import settings
+
+
+def check_and_launch_sagemaker(func):
+    def wrapper(request, *args, **kwargs):
+        sagemaker = boto3.client("sagemaker")
+        endpoint_name = settings.SAGEMAKER_ENDPOINT_NAME
+        if not endpoint_name:
+            return func(request, *args, **kwargs)
```

Comment:
The functions being decorated might not have this signature (almost certainly won't) - can we make it more generic.

Reply:
I've removed the request argument so it's not reliant on django (also removed the reliance on django settings for the same reason)